WorldCupWrapped/Repositories/TeamRepository/TeamRepository.cs

```diff
@@ -28,9 +29,22 @@ public async Task<List<Team>> GetAllTeams()
         {
             return _table.ToList();
         }
-
-        public async Task UpdateTeams()
+        public async Task<List<TrophyCount>> GetTeamTrophies(string teamName)
```

Comment:
si daca stau sa ma gandesc arata mai bine cand ai /Team/Arg/Trophies sus in url

Reply:
da, e fancy asa:))

Reply:
bravo bos